examples/creatives/simid_overlay.js

```diff
@@ -57,4 +58,20 @@ class SimidOverlay extends BaseSimidCreative {
     document.getElementById('log_button').addEventListener(
       'click', sendLogFunction.bind(this));
   }
+
+  sendChangeAdDuration_() {
+    const sendAdDurationFunction = () => {
```

Comment:
These sorts of anonymous function are fine, but most developers would probably prefer to make this a class level function.

I think the reason is that it makes debugging easier.  If there is a stack trace, these will show as anonymous function whereas using a class function will show line numbers.

Reply:
Got it